beeline/src/java/org/apache/hive/beeline/TableOutputFormat.java

```diff
@@ -116,6 +116,10 @@ ColorBuffer getOutputString(Rows rows, Rows.Row row, String delim) {
       if (buf.getVisibleLength() > 0) {
         buf.green(delim);
       }
+ 
+      if (row.values[i] == null) {
```

Comment:
In apache/master, we have "SELECT 'Upgrading MetaStore schema from 2.1.0 to 2.2.0' AS MESSAGE;" which shouldn't have the issue as per description in jira.
I think, skipping the row based on null values is incorrect as it may have any other side effect.

Reply:
The issue is coming only for mysql 5.7 ...no testing till now has been done for mysql 5.7 ..all previous test are done for 5.6. The null check will not create any problem as if the row header is not present , no point color coding the content.